models/ai85net-tinierssd-face.py

```diff
@@ -0,0 +1,435 @@
+###################################################################################################
+#
+# Copyright (C) 2023 Maxim Integrated Products, Inc. All Rights Reserved.
+#
+# Maxim Integrated Products, Inc. Default Copyright Notice:
+# https://www.maximintegrated.com/en/aboutus/legal/copyrights.html
+#
+# detect_objects function is adopted from: GitHub repository:
+# https://github.com/sgrvinod/a-PyTorch-Tutorial-to-Object-Detection:
+# MIT License
+# Copyright (c) 2019 Sagar Vinodababu
+###################################################################################################
+"""
+Tiny SSD (Single Shot Detector) Variant Model for Face Detection
+"""
+from math import sqrt
+
+import torch
+import torch.nn.functional as F
+from torch import nn
+
+import ai8x
+import utils.object_detection_utils as obj_detect_utils
+
+
+class TinySSDBaseFace(nn.Module):
+    """
+    Base convolutions to produce lower-level feature maps.
+    """
+
+    def __init__(self, **kwargs):
+        super().__init__()
+
+        # Standard convolutional layers
+        self.fire1 = ai8x.FusedMaxPoolConv2dReLU(3, 16, 3, padding=1, bias=False)
+        self.fire2 = ai8x.FusedMaxPoolConv2dReLU(16, 32, 3, padding=1, bias=False)
+
+        self.fire3 = ai8x.FusedConv2dBNReLU(32, 64, 3, padding=1, **kwargs)
+        self.fire4 = ai8x.FusedConv2dBNReLU(64, 64, 3, padding=1, **kwargs)
+
+        self.fire5 = ai8x.FusedConv2dBNReLU(64, 64, 3, padding=1, **kwargs)
+
+        self.fire6 = ai8x.FusedConv2dBNReLU(64, 64, 3, padding=1, **kwargs)
+        self.fire7 = ai8x.FusedConv2dBNReLU(64, 128, 3, padding=1, **kwargs)
+        self.fire8 = ai8x.FusedConv2dBNReLU(128, 32, 3, padding=1, **kwargs)
+
+        self.fire9 = ai8x.FusedMaxPoolConv2dBNReLU(32, 32, 3, padding=1,
+                                                   **kwargs)
+
+        self.fire10 = ai8x.FusedMaxPoolConv2dBNReLU(32, 32, 3, padding=1, **kwargs)
+
+    def forward(self, image):

```

Comment:
Could you please arrange the content of the forward method of TinySSDBaseFace class. There too many blank lines.

Reply:
Realized in the new commits.